lib/main.dart

```diff
@@ -63,6 +71,28 @@ class QuimifyApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    // useEffect(() {
```

Comment:
What is this commented code for?

Reply:
It was from a previous attempt I had made but it caused some issues, I'll remove it